mypyc/common.py

```diff
@@ -5,6 +5,7 @@
 STATIC_PREFIX = 'CPyStatic_'  # Static variables (for literals etc.)
 TYPE_PREFIX = 'CPyType_'  # Type object struct
 ATTR_PREFIX = '_'  # Attributes
+PROPSET_PREFIX = 'CPyPropSet_'
```

Comment:
Since this is a name that is going into a dictionary of names on a ClassIR, I would name this something like `__mypyc_setter__`.

(In general though our name mangling is kind of a mess.)